KitePrintSDK/src/main/java/ly/kite/print/PrintJob.java

```diff
@@ -15,29 +17,74 @@
 /**
  * Created by deonbotha on 09/02/2014.
  */
-public abstract class PrintJob implements Parcelable, Serializable {
+public abstract class PrintJob implements Parcelable {
```

Comment:
I'm sure we discussed it but happy to move away from the word `Print` wherever it makes sense as we're very much more about "personalised products" now beyond just traditional print. i.e. `PrintJob` -> `Job`, `PrintOrder` -> `Order`. Not a dependency for merge, you can refactor at your leisure.